config.js

```diff
@@ -18,8 +18,11 @@ exports.create = (api) => {
 
     config = addSockets(config)
     config = fixLocalhost(config)
-    // config = pubHopSettings(config)
-    // config = torOnly(config)
+
+    if (false) {  // TODO figure out how to check if this is being run in/ out of electron
```

Comment:
@mixmix it might be a good idea if these options we're used in ahoy as well, at least if that is running before patchbay. Let me know if you want to have a chat about it. Ahoy looks like a nice abstraction, so keen to help out however I can.